.github/workflows/tethys.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       matrix:
         platform: [ubuntu-latest, macos-latest]
         python-version: ["3.10", "3.11", "3.12"]
-        django-version: ["3.2", "4.2", "5.1"]
+        django-version: ["3.2", "4.2", "5.2"]
```

Comment:
please make this same change in all places where it is applicable. Update  tethys-release.yml too and check for multiple instances in both files.